src/Umbraco.Web.BackOffice/Controllers/MemberController.cs

```diff
@@ -288,172 +308,260 @@ public ActionResult<MemberDisplay> GetEmpty(string contentTypeAlias = null)
+                        object origValue = contentItem.PersistedContent.GetValue(sensitiveProperty.Alias);
                         destProp.Value = origValue;
                     }
                 }
             }
 
-            var isLockedOut = contentItem.IsLockedOut;
+            bool isLockedOut = contentItem.IsLockedOut;
 
-            //if they were locked but now they are trying to be unlocked
+            // if they were locked but now they are trying to be unlocked
             if (contentItem.PersistedContent.IsLockedOut && isLockedOut == false)
             {
                 contentItem.PersistedContent.IsLockedOut = false;
                 contentItem.PersistedContent.FailedPasswordAttempts = 0;
             }
             else if (!contentItem.PersistedContent.IsLockedOut && isLockedOut)
             {
-                //NOTE: This should not ever happen unless someone is mucking around with the request data.
-                //An admin cannot simply lock a user, they get locked out by password attempts, but an admin can un-approve them
+                // NOTE: This should not ever happen unless someone is mucking around with the request data.
+                // An admin cannot simply lock a user, they get locked out by password attempts, but an admin can un-approve them
                 ModelState.AddModelError("custom", "An admin cannot lock a user");
             }
 
-            //no password changes then exit ?
-            if (contentItem.Password == null)
-                return;
+            MembersIdentityUser identityMember = await _memberManager.FindByIdAsync(contentItem.Id.ToString());
+            if (identityMember == null)
+            {
+                return new ValidationErrorResult("Member was not found");
+            }
 
-            throw new NotImplementedException("Members have not been migrated to netcore");
-            // TODO: all member password processing and creation needs to be done with a new aspnet identity MemberUserManager that hasn't been created yet.
-            // set the password
-            //contentItem.PersistedContent.RawPasswordValue = _passwordSecurity.HashPasswordForStorage(contentItem.Password.NewPassword);
-        }
+            if (contentItem.Password != null)
+            {
+                IdentityResult validatePassword = await _memberManager.ValidatePasswordAsync(contentItem.Password.NewPassword);
+                if (validatePassword.Succeeded == false)
+                {
+                    return new ValidationErrorResult(validatePassword.Errors.ToErrorMessage());
+                }
 
-        private static void UpdateName(MemberSave memberSave)
-        {
-            //Don't update the name if it is empty
-            if (memberSave.Name.IsNullOrWhiteSpace() == false)
+                string newPassword = _memberManager.HashPassword(contentItem.Password.NewPassword);
```

Comment:
@Shazwazza need to consider best approach password changing functionality preferences from back-office. Have done this here to hash the password and update the changed date in line 453

Reply:
This is how user passwords are changed: https://github.com/umbraco/Umbraco-CMS/blob/netcore/netcore/src/Umbraco.Web.BackOffice/Controllers/UsersController.cs#L684

In theory we should do something very similar with members instead of just directly re-saving the hash. I suppose this is also why you've exposed `HashPassword` directly? Users does this just based on aspnet identity, creates a reset token and call `ChangePasswordWithResetAsync` with the token. The PasswordChanger class encompasses some of this logic, perhaps that can be shared too or parts re-used.

Reply:
Also the password changing in the UI for users has been changed so it's not part of the same saving request. Members should do this too but that requires UI changes. Though, that should be pretty much the same as what users do to. Maybe that is out of scope but should be done and would simplify somethings.

Reply:
Ok thanks - will take a look

Reply:
@bergmania please can you create a separate ticket for the Members password changing UI changes to match the users changes?

Reply:
@Shazwazza since we already have the method, I will make some changes to reuse the same token functionality that is in `PasswordChanger` and inject it rather than newing it up (as per the TODO comment).

Reply:
I have added an intern task here: https://dev.azure.com/umbraco/D-Team%20Tracker/_workitems/edit/10496

Reply:
@bergmania please can you create a separate task that I can do separately for removing the hashpassword and reusing the PasswordChanger class?

Reply:
Added another intern task here: https://dev.azure.com/umbraco/D-Team%20Tracker/_backlogs/backlog/CMS%20Team/Epics/?workitem=10551

Just let me know if you start on the task :)

Reply:
Fab, thanks @bergmania